.github/workflows/auto-generated-checker.yml

```diff
@@ -0,0 +1,24 @@
+name: Auto-Generated Code Checker
+on:
```

Comment:
Last change...can you make this `on:` section match: https://github.com/ava-labs/hypersdk/blob/580b03c0b4f9b00c95bbd00a699e2c019e09d128/.github/workflows/hypersdk-static-analysis.yml#L6-L10

Reply:
Fixed. Really foolish mistakes, I apologize!

Reply:
No worries! Running now.